src/vs-bicep/Bicep.VSLanguageServerClient.Vsix/Bicep.VSLanguageServerClient.Vsix.csproj

```diff
@@ -61,7 +61,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.VisualStudio.SDK">
-      <Version>17.9.37000</Version>
+      <Version>17.10.40171</Version>
```

Comment:
Curious, how would this affect installation of the vsix on old versions of VS? I'm assuming that scenario is supported but let me know if that isn't.

Reply:
Good question.

Reply:
Will have QA test.  The language server itself is out of proc so not an issue, but the language server client is in-proc.

Reply:
This causes us to require VS 17.10 or higher.  I'm enforcing this in the vsix now.